### DISCUSSION TITLE
Where can I get the details of FT4 and FT8 protocols and decoding/encoding?

### DISCUSSION THREAD

**Discussion by fahadh4ilyas** [kgoba/ft8_lib#28] [Q&A] [answered]
I tried to open https://physics.princeton.edu/pulsar/k1jt/FT4_FT8_QEX.pdf but the link seems already died. May I get the PDF file from another link?

**Comment by kholia** [comment, marked as answer]
Here is a local copy.

[FT4_FT8_QEX.pdf](https://github.com/kgoba/ft8_lib/files/10671717/FT4_FT8_QEX.pdf)

**Reply by fahadh4ilyas** [reply]
Thank you! I tried find it anywhere but to no avail.
